### PR TITLE
Import data

### PR DESCRIPTION
**Pull request by saramcallister**
Added form to home page to allow for importing data. Flash messages appear if the import was successful or if it failed (threw and error).

Default: 
screen shot 2017-11-13 at 6 54 43 pm

After success: 
screen shot 2017-11-13 at 6 54 57 pm

After failure: 
screen shot 2017-11-13 at 6 55 10 pm

### CODE REVIEW

File: teamworkApp/config/routes.rb [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+
+
   root 'high_voltage/pages#show', id: 'home'
+
+  post '/python/importData/' => 'python#importData'
+
   get '/python/studentGraph/' => "python#studentGraph"  
   get '/python/overallBar/' => 'python#overallBar'
-  get '/python/importData/' => 'python#importData'
+  # get '/python/importData/' => 'python#importData'
```
Comment (hmc-cs-mcmurphy) [inline teamworkApp/config/routes.rb]: Why is this commented out?

File: teamworkApp/lib/importData.py [hunk 2]
```diff
@@ -263,10 +263,12 @@ def main():
 		# make sure the path has been properly specified
 		try:
 			execute_insert(filename)
-		except FileNotFoundError:
+		except FileNotFoundError as error:
 			print("File must have an accessible path!")
+			raise error
 	else:
 		print("File must be a CSV with extension .csv!")
+		raise ValueError("Needs a CSV input")
```
Comment (hmc-cs-mcmurphy) [inline teamworkApp/lib/importData.py:271]: Nice additions, thank you!

**Review by hmc-cs-mcmurphy — COMMENTED** [review]
So does this now successfully take in a .csv?

File: teamworkApp/app/controllers/python_controller.rb [hunk 3]
```diff
@@ -10,7 +10,13 @@ def studentGraph
     end
 
     def importData
-        system 'python lib/importData.py'
+        path = params[:file].path
```
Comment (reagan-smith) [inline teamworkApp/app/controllers/python_controller.rb:13]: Nice :D

File: teamworkApp/config/routes.rb (hunk 1 above)
Comment (saramcallister) [reply teamworkApp/config/routes.rb]: That was me, it gets used as a post rather than a get, I will delete the line.

**Pull request merged**